tests/php/test_class-wc-connect-shipping-method.php

```diff
@@ -103,31 +103,6 @@ public function test_is_valid_package_destination( $package, $expected ) {
 	/**
 	 * @covers WC_Connect_Shipping_Method::calculate_shipping
 	 */
-	public function test_invalid_destination_calculate_shipping() {
```

Comment:
It seems to me that you didn't have to completely remove this test because of the removal of the log message. Can you explain?